### PR TITLE
dev: add playwright component tests

### PR DESCRIPTION
**Pull request by mastercactapus**
**Description:**
Adds component testing via `make test-components` that can be used to run tests on react components.

- An initial test for the `TelTextField` was added

### CODE REVIEW

**Review by tony-tvu — DISMISSED** [review]
LGTM

**Review by tony-tvu — APPROVED** [review]

**Review by tony-tvu — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -72,4 +72,11 @@ smoketest_db_dump
...
 /.pnp.*
-
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```
Comment (Forfold) [inline .gitignore:82]:
```suggestion
/test-results/
/playwright-report/
/blob-report/
/playwright/.cache/
```

File: web/src/app/util/TelTextField.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ export default function TelTextField(
   // check validation of the input phoneNumber through graphql
   const [{ data }] = useQuery({
     query: isValidNumber,
-    variables: { number: '+' + phoneNumber },
```
Comment (Forfold) [inline web/src/app/util/TelTextField.tsx:48]: Is the `+` sign w/country code no longer needed?

**Review by Forfold — CHANGES_REQUESTED** [review]

File: web/src/app/util/TelTextField.tsx (hunk 2 above)
Comment (tony-tvu) [reply web/src/app/util/TelTextField.tsx:48]: This was a bug we found that currently exists in production. We're already appending a `+` on line 99 in the `handleChange` func, so it's been getting sent back to API as `++15555555`.

**Review by tony-tvu — APPROVED** [review]

**Review by Forfold — APPROVED** [review]

**Pull request merged**